Specs/Scene/ShadowMapSpec.js

```diff
@@ -797,7 +827,17 @@ describe(
       scene.light = new DirectionalLight({
         direction: lightDirectionAngle,
       });
-      renderAndExpect(unshadowedColor);
+
+      // Change the time so that the shadows are no longer pointing straight down
+      renderAndCall(function (rgba) {
+        // Due to shading differences, the result might not exactly the same, but
+        // it should be close
+        const [r, g, b, a] = colorAbsoluteDifference(rgba, unshadowedColor);
+        expect(r).toBeLessThan(48);
+        expect(g).toBeLessThan(4);
```

Comment:
The epsilons seem arbitrary to me since they differ so much for each test... is there no way to make them more consistent? Why do we consider a color difference of 48 or even 64 "close" ?